### PR TITLE
Build faq page

### LINKED ISSUE #617: [c.c application page] Build FAQ page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

Build an FAQ page on `/careers/application/faq`
Demo at https://canonical-com-619.demos.haus/careers/application/faq

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare page against [copy doc](https://docs.google.com/document/d/12_O_QtPqfv9p7kaDFdiP8Xq-kY5xa1XB-C-CRbXjDew/edit) and [design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=5916%3A8133)

## Issue / Card

Fixes #617 

## Screenshots

Publisher of Ubuntu | Canonical ()

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
Looks great thanks @minkyngkm

File: templates/careers/application/faq.html [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+{% block extra_metatags %}
+  <meta name="robots" content="noindex">
+{% endblock %}
+
+{% block content %}
+ 
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">FAQ</h2>
```
Comment (sowasred2012) [inline templates/careers/application/faq.html:11]: This is the first heading on the page, so should be `h1` - and the `h3`s below should be bumped up to `h2`
```suggestion
    <h1 class="p-text--x-small-capitalised u-align-text--x-small-to-default">FAQ</h1>
```

**Review by sowasred2012 — COMMENTED** [review]
Nice work! Just one minor change and this will be good to go

**Comment by sowasred2012** [comment]
@minkyngkm still also need to update the `h3` tags to `h2`:

image

**Comment by minkyngkm** [comment]
ah sorry, done now! Thanks @sowasred2012 🙌

**Review by sowasred2012 — APPROVED** [review]
noice

File: static/sass/_pattern_application.scss [hunk 2]
```diff
@@ -87,8 +87,9 @@
         }
       }
     }
-    
-    .p-accordion__tab {
+    // TODO : the accordion component needs to be fixed in vanilla. The issue was filed on https://github.com/canonical/vanilla-framework/issues/4563. This styles written locally needs to be removed after vanilla update.
+    .p-accordion__tab span {
```
Comment (lyubomir-popov) [inline static/sass/_pattern_application.scss:91]: I would put a class like u-default-max-width. We shouldn't use element selectors.

**Review by lyubomir-popov — CHANGES_REQUESTED** [review]

File: static/sass/_pattern_application.scss (hunk 2 above)
Comment (minkyngkm) [reply static/sass/_pattern_application.scss]: Thanks @lyubomir-popov. Fixed!

**Pull request merged**